staging/src/k8s.io/apiserver/pkg/cel/formats.go

```diff
@@ -0,0 +1,54 @@
+package cel
```

Comment:
We need to add runtime and estimated costs for these functions and tests.  I think we can assume format checks have the same time complexity as RE2 regex?

Reply:
Added. Used same impl from regex lib. Unfortuantely not easy to access underlying regex so I used a max regex string size of 128